cpp/velox/compute/VeloxPlanConverter.cc

```diff
@@ -587,6 +598,25 @@ class VeloxPlanConverter::WholeStageResIterFirstStage
   std::vector<core::PlanNodeId> streamIds_;
   std::vector<std::vector<exec::Split>> splits_;
   bool noMoreSplits_ = false;
+
+  // Extract the partition column and value from a path of split.
+  // Extraction for multiple partition columns is not supported.
```

Comment:
Multiple partition columns is a common scene, it's better to throw unsupported exception. What will happen if we meet this case now?

Reply:
I guess the case you refer to is like `/my_dataset/year=2022/month=July`. I can support this case as weill.

Reply:
Fixed.